W-11894579 document dynamic log levels (#872)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/deploy-to-runtime-fabric.adoc

. In the *New Value* field, add a value.
. Add additional properties as needed.

=== Configure Logging and Log Levels

> Anypoint Monitoring provides access to log data for applications deployed to Rutime Fabric. To access logs with Anypoint Monitoring, you need a https://www.mulesoft.com/anypoint-pricing[Titanium subscription].

Reviewer: ```suggestion
Anypoint Monitoring provides access to log data for applications deployed to Runtime Fabric. To access logs with Anypoint Monitoring, you need a https://www.mulesoft.com/anypoint-pricing[Titanium subscription].
```

. Click the *Logging* tab.
. Select the *Enable application logs* option.

> . To configure additional log levels, use the drop-down list to select the log level:

Reviewer: ```suggestion
. Select the log level from the drop-down list:
```
